src/services/FileService.ts

```diff
@@ -0,0 +1,271 @@
+import { File } from "../entities/File";
+import MinioService from "./MinioService";
+import { initORM } from "../db";
+import { Page, toPageDTO } from "../utils/pagination";
+
+class FileService {
+  private minioService: MinioService | null = null;
+  private bucketName: string;
+
+  constructor() {
+    this.bucketName = 'ghost-drive'; // Default bucket name
```

Comment:
Please read again README.md https://github.com/codingcat0405/ghost-drive-be/blob/master/README.md#-file-system-architecture to review the file system architecture of the project. Each user have his own bucket name (which have been created when user register check `src/services/UserServices.ts` The default minio bucket just for common upload api so we can saved user public asset for frontend like user avatar,....